content/applications/finance/accounting/payments/internal_transfers.rst

```diff
@@ -2,45 +2,40 @@
 Internal transfers
 ==================
 
-In Odoo, internal money transfers can be made in a few clicks. You need at least either two bank
-accounts, two cash journals, or one bank account and one cash journal.
+Internal money transfers can be handled in Odoo. At least two bank accounts are needed to make
+internal transfers.
+
+.. seealso::
+   :doc:`How to add an additional bank account <../bank>`
 
 Configuration
 =============
 
 An internal transfer account is automatically created on your database based on your company's
 localization and depending on your country’s legislation. If needed, the default :guilabel:`Internal
-transfer account` can be modified by going to :menuselection:`Accounting app --> Configuration -->
-Settings` and then under the :guilabel:`Default Accounts` section.
-
-.. note::
-   At least two bank accounts are needed to make internal transfers. Refer to
-   :doc:`Bank and cash accounts section <../bank>` to see how to add an additional bank
-   account to your database.
+transfer account` can be modified by going to :menuselection:`Accounting --> Configuration -->
+Settings --> Default Accounts section`.
 
 Register an internal transfer from one bank to another
 ======================================================
 
-Let's say you have two bank accounts registered on your database and you want to transfer 1,000 USD
-from Bank A to Bank B.
-
 Log an internal transfer
```

Comment:
See my comment on the reconciliation section, but if you remove the reconciliation section of this doc, I would remove this sub-title as well as it would not be necessary.